mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/options/AuthenticationMechanismType.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.options;
+
+/**
+ * Available authentication methods for MongoDB.
+ *
+ * @see com.mongodb.MongoCredential
+ * @author Davide D'Alto
+ */
+public enum AuthenticationMechanismType {
+
+	GSSAPI("GSSAPI"),
+	MONGODB_CR("MONGODB-CR"),
```

Comment:
That's the value used by mongodb so it has to be `MONGO-CR` note that MONGODB_CR will work anyway.

Now that I think about it it should be `MongoCredential.MONGODB_CR_MECHANISM`